conf/base.config

```diff
@@ -41,5 +39,12 @@ process {
   withName:get_software_versions {
     cache = false
   }
-
+  withName: GetSRA {
+    cpus = { check_max (2, 'cpus')}
+    errorStrategy = { task.attempt <= 30 ? 'retry' : 'ignore' }
+    container = 'avsastry/fasterq-dump:1.0'
+  }
```

Comment:
ToDo:
- [ ] remove the container, https://github.com/TheJacksonLaboratory/atacseq/commit/2d7cd80802f260aab1de832f8fd063a60a1e184d adds the dependencies in the main nf-core container.